src/components/SnackBoxCard.tsx

```diff
@@ -56,15 +90,15 @@ export default function ProductCard({
           <p
             className={`text-black truncate text-${fontSize} font-normal max-w-full`}
           >
-            {name}
+            {item.name}
           </p>
 
           {size === "sm" ? (
             <div className="flex flex-row w-full justify-between items-center ">
               <p
                 className={`text-secondaryT-main text-${fontSize} font-semibold`}
               >
-                ฿{price.toFixed(2)}
+                ฿{item.price?.toFixed(2)}
```

Comment:
I think this should be item?.price.toFixed(2) || 0, like if the price is null

Reply:
ok